app/mailers/merge_mailer.rb

```diff
@@ -3,9 +3,11 @@ class MergeMailer < ActionMailer::Base
 
   default :from => Proc.new { Settings.sender_email_address }
 
-  def merge_successful(build, emails, stdout_and_stderr)
+  def merge_successful(build, merged_ref, emails, stdout_and_stderr)
```

Comment:
technically this variable should be called `merge_ref`. The merged_ref is the ancestor commit of the merge_ref.